<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartRemoteDialog</class>
 <widget class="QDialog" name="StartRemoteDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Start Debugger</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="debuggerLabel">
       <property name="text">
        <string>&amp;Debugger:</string>
       </property>
       <property name="buddy">
        <cstring>debuggerPathChooser</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="Utils::PathChooser" name="debuggerPathChooser"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="executableLabel">
       <property name="text">
        <string>Local &amp;executable:</string>
       </property>
       <property name="buddy">
        <cstring>executablePathChooser</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="Utils::PathChooser" name="executablePathChooser"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="channelLabel">
       <property name="text">
        <string>&amp;Host and port:</string>
       </property>
       <property name="buddy">
        <cstring>channelLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="channelLineEdit">
       <property name="text">
        <string notr="true">localhost:5115</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="architectureLabel">
       <property name="text">
        <string>&amp;Architecture:</string>
       </property>
       <property name="buddy">
        <cstring>architectureComboBox</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="architectureComboBox">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="gnuTargetLabel">
       <property name="text">
        <string>&amp;GNU target</string>
       </property>
       <property name="buddy">
        <cstring>gnuTargetComboBox</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="gnuTargetComboBox">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="sysrootLabel">
       <property name="text">
        <string>Sys&amp;root:</string>
       </property>
       <property name="buddy">
        <cstring>sysrootPathChooser</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="Utils::PathChooser" name="sysrootPathChooser"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="useServerStartScriptLabel">
       <property name="text">
        <string>&amp;Use server start script:</string>
       </property>
       <property name="buddy">
        <cstring>useServerStartScriptCheckBox</cstring>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QCheckBox" name="useServerStartScriptCheckBox"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="serverStartScriptLabel">
       <property name="text">
        <string>Server st&amp;art script:</string>
       </property>
       <property name="buddy">
        <cstring>serverStartScript</cstring>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="Utils::PathChooser" name="serverStartScript"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::PathChooser</class>
   <extends>QWidget</extends>
   <header location="global">utils/pathchooser.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
